source/main.py

```diff
@@ -1,7 +1,8 @@
-from typing import Dict, Union
+from typing import Dict
 
 import menus
 from blessed import Terminal
```

Comment:
```suggestion
from blessed import Interface
```
We use Interface instead of Terminal